platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ImportMigrationConfigurationAdminContext.java

```diff
@@ -125,7 +126,7 @@ public ImportMigrationConfigurationAdminContext(
     context.getReport().doAfterCompletion(this::deleteUnexportedConfigurationsAfterCompletion);
   }
 
-  public Stream<ImportMigrationConfigurationAdminEntry> entries() {
+  public Stream<ImportMigrationConfigurationAdminEntry> memoryEntries() {
```

Comment:
❓Shouldn't we need to update some unit tests since the method name has changed?

Reply:
All tests done for this migratable are higher level than unit tests per agreement on the first PR.